examples/README.md

```diff
@@ -606,6 +406,232 @@ The blueprint contains the following:
 
 [batch-mpi.yaml]: ../examples/batch-mpi.yaml
 
+### [lustre.yaml] ![core-badge]
+
+Creates a DDN EXAScaler lustre file-system that is mounted in two client instances.
+
+The [DDN Exascaler Lustre](../community/modules/file-system/DDN-EXAScaler/README.md)
+file system is designed for high IO performance. It has a capacity of ~10TiB and is mounted at `/lustre`.
```

Comment:
nit: `default` capacity? You can set to higher correct?

Reply:
yup, fixed.